.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
 npm-debug.log
 docker-modem/
+.project
```

Comment:
Yup I like to keep public .gitignore as standard as possible we never know what other use cases/setups/... are out there.
Could you please fix this?